pymoto/modules/assembly.py

```diff
@@ -213,7 +213,7 @@ def _prepare(self, domain: DomainDefinition, *args,
 
         # Numerical integration
         siz = domain.element_size
-        w = np.prod(siz[:domain.dim]/2)
+        w = np.prod(siz/2)
```

Comment:
I think the original is the correct one. Can you verify this with the test I've added?

Reply:
Verified and restored original